### DISCUSSION TITLE
'No Audio' option for feeds

### DISCUSSION THREAD

**Discussion by tonogdlp** [f1multiviewer/issue-tracker#12] [Feature requests]
Even if it is minimal, audio consumes bandwith even when muted. Can you add a "No Audio" option for those feeds that will never need audio?

**Comment by jeppeolesen** [comment]
Hey there @tonogdlp 

The video and audio feeds are all coming directly from F1 TV. The only option we have to strip out an audio track from the stream is by muting it.
